src/tech/index.md

```diff
@@ -169,10 +169,9 @@ where we describe what changed and why we changed it. We keep track of any
 future changes we may want to make or think about in the
 [backlog](https://docs.google.com/document/d/16pZuxt_CWp2NFaYaqBIZvuHmPeNWcqKycLkx5ciErbw/edit).
 
-We have a [documented promotion process](/guides/tech-team-promotion-process)
-based on that framework. We designed it to help make fair decisions about
-promotions that reduce the impact any one person can have on the decision as
-much as possible.
+We have a [documented promotion process]navipromotion-process) based on that
```

Comment:
This link is broken (maybe we could use the [`mdl`](https://github.com/markdownlint/markdownlint) gem to lint Markdown? It's quite easy to [configure](https://gitlab.com/beautifulcanoe/peopleops/docs.beautifulcanoe.com/-/blob/main/.mdl.rb)

Reply:
We dropped all linting because it gets in the way of non-technologist editors doing their thing. This is a typo, though.

Reply:
Fixed the typo.